docs/source/reference/core/debug.rst

```diff
@@ -6,20 +6,42 @@ Debug mode
 In debug mode, Chainer checks values of variables on runtime and shows more
 detailed error messages.
 It helps you to debug your programs.
-Instead it requires additional overhead time.
+However, it requires some additional overhead time.
 
-In debug mode, Chainer checks all results of forward and backward computation, and if it founds a NaN value, it raises :class:`RuntimeError`.
+You can enable debug mode with :func:`chainer.using_config`:
+
+.. testcode::
+
+    with chainer.using_config('debug', True):
+       ...
```

Comment:
I can't understand why this code pass the test.

Reply:
It seems `...` is treated as `Ellipsis` by Python interpreter.
Therefore, `...` is completely valid Python code.

Reply:
oh...